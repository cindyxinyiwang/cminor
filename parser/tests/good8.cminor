// special printing

printarray: function void ( a: array [] integer, size: integer ) = {
	str: string = "hello \n world";
	c: char = '\n';
	a: char = '\t';
}