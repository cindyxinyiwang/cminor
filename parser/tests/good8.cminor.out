printarray: function void ( a: array [] integer, size: integer ) = {
	str: string = "hello \n world";
	c: char = '\n';
	a: char = '\t';
} : function void(a: array [] integer, size: integer ) = {
	str: string = "hello \n world";
	c: char = '\n';
	a: char = '\t';
}: array [] integer,: integer)= {
	str: string = "hello \n world"; : string="hello \n world";
	c: char = '\n'; : character='\n';
	a: char = '\t'; : character='t';

}
